src/Components/Nav/Nav.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from "react";
+import "./Nav.scss";
+import NavDropdown from "./NavDropdown";
+
+export default class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      menus: [],
+      activeId: false,
+      isLogin: false,
+      click: false
+    };
+  }
```

Comment:
```suggestion
    state = {
      menus: [],
      activeId: false,
      isLogin: false,
      click: false
    };
  }
```
한나님! constructor안에서 state를 정의해주지 않고 위와 같이 해줄 수 도 있습니다! 
이제 리액트 라이프 사이클을 잘 이해하고 계신 것 같으므로...팁 하나 날리겠습니다 ..💌

Reply:
네네네😊😊😊